reports/discussion.tex

```diff
@@ -4,7 +4,7 @@
 various perspectives. Initially, we developed a direct translation of the
 protocol's Python specification into \tlap{}, but this approach proved
 unsatisfactory due to the reliance on recursion. To address this, we modified
-the specification to use folds in place of recursion, theoretically enabling
+the specification to use folds\rs{if possible, quickly explain what folds are or provide a reference} in place of recursion, theoretically enabling
```

Comment:
I think the whole paragraph doesn't make a lot of sense here.
It was intended to go at the end of the document but got moved.
Moving it back to the end of the text in #94

Reply:
```suggestion
the specification to use folds in place of recursion, theoretically enabling
```